substrate/frame/identity/src/lib.rs

```diff
@@ -129,10 +134,9 @@ pub mod pallet {
 		#[pallet::constant]
 		type MaxSubAccounts: Get<u32>;
 
-		/// Maximum number of additional fields that may be stored in an ID. Needed to bound the I/O
-		/// required to access an identity, but can be pretty high.
+		/// The maximum encoded size of an identity information.
 		#[pallet::constant]
-		type MaxAdditionalFields: Get<u32>;
+		type MaxIdentitySize: Get<u32>;
```

Comment:
Not sure we need this at all? Why have an upper bound here, if this upper bound is already implicitly declared by the `IdentityInformation`.

I don't think we need this.

Reply:
A chain that has 10 different identity fields, should also support that all of them are set.

Reply:
Makes sense. It's gone.